### ISSUE TITLE
Migration plan wizard - validation/results step enhancements

### ISSUE THREAD

**Issue by vconzola** [konveyor/mig-ui#588]
Enlarge icon, center Close button, and remove "box within a box". See [mockup.](https://marvelapp.com/project/4322507/screen/61033371)

If warnings are required, they should be shown as inline notifications.

**Comment by vconzola** [comment]
@mturley When you get to this one , ask me.

**Comment by mturley** [comment]
@ibolton336 I didn't realize I was still assigned here. Are you planning to address this as part of https://github.com/konveyor/mig-ui/pull/807 or do you want it to be a separate PR?

**Comment by ibolton336** [comment]
Ah I think I hit almost everything on this. Maybe you could go over it again once it is merged?

**Comment by mturley** [comment]
Sounds good.
